crates/ruff/src/rules/flake8_tidy_imports/relative_imports.rs

```diff
@@ -22,7 +30,7 @@ pub enum Strictness {
 define_violation!(
     pub struct RelativeImports(pub Strictness);
 );
-impl Violation for RelativeImports {
+impl AlwaysAutofixableViolation for RelativeImports {
```

Comment:
I think this should be a "sometimes" autofixable. You can grep for `Availability::Sometimes` to see examples of that pattern.